Source: https://gitlab.gnome.org/GNOME/libgda/-/merge_requests/208

From: =?utf-8?q?Ahmed_Ba=C3=AFzid?= <ahmed.baizid.0@gmail.com>
Date: Sat, 31 Aug 2024 22:18:56 +0100
Subject: error: function declaration without prototype

Function declaration with no prototype are forbidden in C23.

See: https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2841.htm

Bug: https://gitlab.gnome.org/GNOME/libgda/-/issues/272
Forwarded: https://gitlab.gnome.org/GNOME/libgda/-/merge_requests/208
Bug-Debian: https://bugs.debian.org/1097190
---
 libgda/sql-parser/lemon.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/libgda/sql-parser/lemon.c b/libgda/sql-parser/lemon.c
index afab40f..53adb1f 100644
--- a/libgda/sql-parser/lemon.c
+++ b/libgda/sql-parser/lemon.c
@@ -69,14 +69,6 @@ struct action;
 static struct action *Action_new(void);
 static struct action *Action_sort(struct action *);
 
-/********** From the file "build.h" ************************************/
-void FindRulePrecedences();
-void FindFirstSets();
-void FindStates();
-void FindLinks();
-void FindFollowSets();
-void FindActions();
-
 /********* From the file "configlist.h" *********************************/
 void Configlist_init(void);
 struct config *Configlist_add(struct rule *, int);
